src/js/genetic/genetic.js

```diff
@@ -0,0 +1,151 @@
+let totalPopulation;
+let numOfGeneration;
+let numOfBestGene;
+let currBest;
+let population;
+let bestGene;
+
+function swap(arr, i, j) {
+    let temp = arr[i];
+    arr[i] = arr[j];
+    arr[j] = temp;
+
+    return arr.slice();
+}
+
+function shuffle(arr) {
```

Comment:
Аналогичная функция находится в main.js
```suggestion
function shuffle(array) {
    let currentIndex = array.length, randomIndex;

    while (currentIndex !== 0) {

        randomIndex = Math.floor(Math.random() * currentIndex);
        currentIndex--;
        [array[currentIndex], array[randomIndex]] = [array[randomIndex], array[currentIndex]];
    }

    return array;
}
```